content/events/decentralize-ai-day.md

```diff
@@ -0,0 +1,19 @@
+ventName: "Decentralize AI Day"
+eventDescription: "DeAI Day at ETH Denver 2024." 
+eventTopic: 
+eventType: "Summit" 
```

Comment:
@laeccles The eventType can have one among the following values: Conference, Virtual, Social. Please select one from the above.

Reply:
@WinsCodes I have updated this!